common/lib/capa/capa/inputtypes.py

```diff
@@ -326,11 +326,16 @@ def _get_render_context(self):
         }
 
         # Don't add aria-describedby attribute if there are no descriptions
```

Comment:
Comment needs to be updated.

Nit: instead of an if/else, how about a variable that has the description_ids, pre-populated with the status_id? Then you just conditionally add on to it.